src/site/content/en/handbook/grammar/index.md

```diff
@@ -64,9 +70,9 @@ Don't include preceding articles (_a_, _an_, _the_) or surrounding punctuation i
 
 {% endCompare %}
 
-When referring to webpages, either on web.dev or elsewhere, hyperlink the webpage title and put double quotation marks outside the hyperlink. (See example above.)
+When referring to webpages, either on web.dev or elsewhere, hyperlink the webpage title and put double quotation marks outside the hyperlink.
```

Comment:
Why should they add a double quotation? I see this inconsistently applied across the docs and I don't see why the quotation is needed or helpful. Seems like an extra hangup that people will forget to add, and it doesn't add anything for readers (it's already visibly a hyperlink).

Reply:
As we discussed, this guidance was based on a misleading example in the GDDSC. I've updated this langauge and fixed link punctuation throughout the site in a [separate PR](https://github.com/GoogleChrome/web.dev/pull/1179).